test/hermes_test/test_cli.py

```diff
@@ -1,7 +1,7 @@
 # SPDX-FileCopyrightText: 2022 German Aerospace Center (DLR)
 #
 # SPDX-License-Identifier: Apache-2.0
-
+import pathlib
```

Comment:
Automatic import by IDE? :-) Should be below the SPDX-FileContributor

Reply:
Yes, indeed...